docs/source/concepts/ledger-model/ledger-structure.rst

```diff
@@ -86,6 +88,7 @@ underlying type of parties.
    Action       ::= 'Create' contract
                     | 'Exercise' party* contract Kind Transaction
                     | 'Fetch' party* contract
+                    | 'NoKey' key
```

Comment:
Why is this NoKey? It's not clear to me what this means.

Reply:
Perhaps `NoSuchKey`, or `IsFree`? I know it might be painful to rename now.

Reply:
I like `NoSuchKey`. Renamed. `IsFree` seems too general as it does not refer to a key.